man/systemd.network.xml

```diff
@@ -1099,6 +1099,13 @@ IPv6Token=prefixstable:2002:da8:1::</programlisting></para>
             Defaults to <literal>global</literal>.</para>
           </listitem>
         </varlistentry>
+        <varlistentry>
+          <term><varname>RouteMetric=</varname></term>
+          <listitem>
+            <para>The metric of the prefix route. Takes an unsigned integer in the range 0…4294967295.
```

Comment:
maybe explain what the prefix route is, might not be a term everyone groks, i.e. add "(the route pointing to the subnet of the configured IP addresses', taking the configured subnet mask into account)" or someting like that